<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏后自动最小化</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>快捷键打开</string>
   </property>
  </widget>
  <widget class="QSlider" name="transparent_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="sure">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QSlider" name="transparent_input">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="transparent_output">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_output">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>输出框透明度:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_input">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>输入框透明度:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>按钮的透明度:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏前询问游戏分辨率</string>
   </property>
  </widget>
  <widget class="QSlider" name="transparent_window">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_window">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>程序的透明度:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏后启动器自动靠右</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>自动检查更新版本</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
